src/ui/components/InitiativesEditor.js

```diff
@@ -0,0 +1,74 @@
+// @flow
+
+import React from "react";
+import {Admin, Resource, Loading} from "react-admin";
+import {
+  InitiativeList,
+  InitiativeCreate,
+  InitiativeEdit,
+} from "../components/Initiatives";
```

Comment:
Can shorten this import
```suggestion
} from "../Initiatives";
```

Reply:
addressed this in #1968